pkg/systemd/services/service-details.jsx

```diff
@@ -447,13 +455,37 @@ export class ServiceDetails extends React.Component {
             { Name: _("Before"), Units: this.props.unit.Before },
             { Name: _("After"), Units: this.props.unit.After },
             { Name: _("On failure"), Units: this.props.unit.OnFailure },
-            { Name: _("Triggers"), Units: this.props.unit.Triggers },
-            { Name: _("Triggered by"), Units: this.props.unit.TriggeredBy },
             { Name: _("Propagates reload to"), Units: this.props.unit.PropagatesReloadTo },
             { Name: _("Reload propagated from"), Units: this.props.unit.ReloadPropagatedFrom },
             { Name: _("Joins namespace of"), Units: this.props.unit.JoinsNamespaceOf }
         ];
 
+        const relationshipsToList = rels => {
+            return rels.filter(rel => rel.Units && rel.Units.length > 0)
+                    .map(rel => {
+                        return (
+                            <DescriptionListGroup key={rel.Name.split().join("")}>
+                                <DescriptionListTerm>{rel.Name}</DescriptionListTerm>
+                                <DescriptionListDescription id={rel.Name.split(" ").join("")}>
+                                    <ul className="comma-list">
+                                        {rel.Units.map(unit => <li key={unit}><a href={"#/" + unit} className={this.props.isValid(unit) ? "" : "disabled"}>{unit}</a></li>)}
```

Comment:
I thought we moved away from `disabled` classes and use the proper `disabled` attribute. I know this is just moving existing code, but it is now the *only* place in cockpit that uses a "disabled" class. If there's a good reason for that, we can look at that in a follow-up, otherwise this may be a good time to clean this up?

Reply:
Sure I will add a commit on top/

Reply:
This is implemented now: services: replace custom disabled class in service relationships comma list